specifyweb/stored_queries/queryfieldspec.py

```diff
@@ -179,6 +179,13 @@ def is_temporal(self):
         field = self.get_field()
         return field is not None and field.is_temporal()
 
+    def is_json(self):
+        field = self.get_field()
+        is_json_field = field is not None and field.type == 'json'
+        if field.sqlalchemy_type is not None: # hasattr(field, 'sqlalchemy_type') and
+            is_json_field = field.sqlalchemy_type.lower() == 'json'
+        return is_json_field
```

Comment:
Hmm why do you need sqlalchemy_type?

Reply:
I initially added it to allow for a field to be 'json' in Django, and 'text' in sql.  I think this might be useful in the future, but if we end up not using it ever, then I'll remove it.